src/pages/NewCarsList/NewCarsList.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from "react";
```

Comment:
If I am assuming correctly we have simply copy pasted the code for All cars to New CarsList and Used Cars List, please do not do this instead just simply modify the existing listing page